### ISSUE TITLE
Compiling instructions for Windows

### ISSUE THREAD

**Issue by avrdude-t** [eventhorizon5/skyscraper#12]
Could you write compiling instructions for Windows? I would like to develop on and for Windows but I've been unsuccessfull in trying to compile Skyscrapersim.

**Comment by eventhorizon5** [comment]
Sure, I just need to make sure the toolchain is together and updated.  I'm glad you want to develop.

**Comment by eventhorizon5** [comment]
I wrote up instructions on how to build it.  Try this out and see if it works for you, I didn't test it myself but it should work.  The Windows section is at the bottom.
https://github.com/eventhorizon5/skyscraper/blob/master/Compiling.md

**Comment by avrdude-t** [comment]
I followed your instructions. At first I got a bunch of errors that said that Ogre couldn't be found. I looked in the additional include directories and discovered there was another env variable "OGRE_HOME_64" that you forgot to mention.
After adding that variable and pointing it to C:\ogre-13.4.3\SDK and building a Release build it worked just fine. Building a debug build still doesn't work though.
